engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -45,32 +97,56 @@ object WorkflowExecutionActor {
   case object WorkflowExecutionAbortedResponse extends WorkflowExecutionActorResponse
   final case class WorkflowExecutionFailedResponse(reasons: Seq[Throwable]) extends WorkflowExecutionActorResponse
 
+  /**
+    * Captures the `ExecutionStoreKey`/`ExecutionStatus` combination for a call that has been created or started
```

Comment:
It is being used in the `startRunnableCalls` but I guess and I can do without it just as well. I'll remove this.